code/__DEFINES/cyborg_traits.dm

```diff
@@ -0,0 +1,7 @@
+#define CYBORG_TRAIT_FLASH_RESISTANT 0x1 // admin only
+#define CYBORG_TRAIT_CLEANING_WALK 0x2 // janitor module
+#define CYBORG_TRAIT_DEFLECTIVE_BALLISTIC_ARMOR 0x3 // admin only
+#define CYBORG_TRAIT_SPEED_BOOSTED 0x4 // not really used but could increase robot dodge in the future  . added by vtec module
+#define CYBORG_TRAIT_AI_HACKED 0x5 // added when a cyborg is hacked by an AI, prevents hacking by other AI's
+#define CYBORG_TRAIT_EMAGGED 0x6 // added when emagged
+#define CYBORG_TRAIT_POWER_SAVER 0x7// not really used but could in the future  . added by power saver module
```

Comment:
This won't work. Looking at the rest of the code, these are supposed to be bitflags.
However, if you are doing bitflags, you need to have the values of 0x1, 0x2, 0x4, 0x8.
The way it is currently coded, the Ballistic Armor trait will also make the borgs clean when walking and being flash resistant.
The AI Hack trait would enable the flash resistant trait, while the emag trait would enable cleaning walk trait.
The power Saver trait would enable flash resistance, cleaning walk and the speed boost.

In this situation, if you have a borg with the AI Hack trait, and remove the flash resistant, you end up with only the speed boost.
And if you have a borg that get flash resistance, then get the cleaning walk, then they suddenly get the Ballistic armor.